backend/Watchdog.Core/Watchdog.Core.API/Controllers/IssuesController.cs

```diff
@@ -42,8 +42,8 @@ public async Task<ActionResult<IssueDto>> GetIssueByIdAsync(int issueId)
         public async Task<ActionResult> GetIssuesInfoLazyAsync(int memberId, [FromBody] FilterModel filterModel,
             [FromQuery] IssueStatus? status)
         {
-            var (issues, totalRecord) = await _issueService.GetIssuesInfoLazyAsync(memberId, filterModel, status);
-            return Ok(new { Collection = issues, TotalRecords = totalRecord });
+            var (issues, totalRecord, counts) = await _issueService.GetIssuesInfoLazyAsync(memberId, filterModel, status);
```

Comment:
Now you return counters in every table change?

Reply:
Yes

Reply:
Okey, already moved into separate request